scripts/docker/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2024 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shell script for running Cloud Orchestrator based on Docker
+
+# Specify docker API version for running cloud orchestrator
+docker_api_version_code=1.43
+docker_api_version_installed=$(docker version --format '{{.Client.APIVersion}}')
+docker_api_version=$(\
+  echo -e $docker_api_version_code\\n$docker_api_version_installed \
+  | sort --version-sort \
+  | head -n 1)
+
+dir=$(dirname $0)
+pushd $dir > /dev/null
+
```

Comment:
just curious: why we need this?

It seems to me that we can successfully get `cloud_android_orchestration_root` anywhere under the cloud-android-orchestration checkout directory.

Reply:
I added that because of a small difference. I thought that one would make the shell script more stable.

- Without `pushd $(dirname $0)`, `cloud_android_orchestration_root` is findable when current dir is under `cloud-android-orchestration` directory.
- With `pushd $(dirname $0)`, `cloud_android_orchestration_root` is findable even the outside of `cloud-android-orchestration` directory.

Reply:
I see. That's a good point!